components/atom/button/src/index.js

```diff
@@ -213,6 +218,10 @@ AtomButton.propTypes = {
    * Size of button 'small' (default), 'large'
    */
   size: PropTypes.oneOf(Object.values(SIZES)),
+  /**
+   * Align text 'center' (default), 'left'
```

Comment:
maybe it is more accurate to say that this prop aligns the content (not only the text) because icons are also affected by this alignment. 👍🏽